docs/content/manual/release-specific/v1.1.0/rwx_feature.md

```diff
@@ -0,0 +1,322 @@
+8. Verify the longhorn volume, it should reflect the correct size.
+
+# Scale up StatefulSet/Deployment with one pod attached with volume in RWX mode.
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 1 pod.
+4. Write some data in the pod and verify the same data reflects in the ShareManager.
+5. Scale up the StatefulSet/Deployment.
+6. Verify a new volume gets created.
+7. Write some data in the new pod, the same should be reflected in the ShareManager pod.
+8. Verify the longhorn volume, it should reflect the correct size.
+
+# Scale down StatefulSet/Deployment attached with volume in RWX mode to zero.
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 1 pod.
+3. Write some data in the pod and verify the same data reflects in the ShareManager.
+4. Scale down the StatefulSet/Deployment to zero
+5. Verify the ShareManager pod gets deleted.
+6. Verify the volume should be in detached state.
+7. Create a new StatefulSet/Deployment with the existing PVC.
+8. Verify the ShareManager should get created and volume should become attached.
+9. Verify the data.
+10. Delete the newly created StatefulSet/Deployment.
+11. Verify the ShareManager pod gets deleted again.
+12. Scale up the first StatefulSet/Deployment.
+13. Verify the ShareManager should get created and volume should become attached.
+14. Verify the data.
+
+# Delete the Workload StatefulSet/Deployment attached with RWX volume.
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 2 pods.
+3. Write some data in the pod and verify the same data reflects in the ShareManager.
+4. Delete the workload.
+5. Verify the ShareManager pod gets deleted but the CRD should not be deleted.
+6. Verify the ShareManager.status.state == "stopped". `kubectl get ShareManager -n longhorn-system`
+6. Verify the volume should be in detached state.
+7. Create another StatefulSet with existing PVC.
+8. Verify the ShareManager should get created and volume should become attached.
+9. Verify the data.
+
+# Take snapshot and backup of a RWX volume in Longhorn.
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 2 pods.
+3. Write some data in the pod.
+4. Take a snapshot and a backup.
+5. Write some more data into the pod.
+6. Revert to snapshot 1 and verify the data.
+
+# Restore a backup taken from a RWX volume in Longhorn.
+1. Create a PVC with RWX mode using longhorn class by selecting the option `read write many`.
+2. Attach the PVC to a StatefulSet/Deployment with 2 pods.
+3. Write some data in the pod.
+4. Take a backup of the RWX volume.
+5. Restore from the backup.
```

Comment:
user specifies the access mode, when requesting the restore.

Reply:
Corrected.